### PR TITLE
feat!: execute_mode default to online

### PR DESCRIPTION
**Pull request by aceforeverd**

### CODE REVIEW

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Comment by vagetablechicken** [comment]
We always use default offline mode in docs demo, all of them should be updated, let's create a new issue for it? Or fix in this pr

**Comment by aceforeverd** [comment]
> We always use default offline mode in docs demo, all of them should be updated, let's create a new issue for it? Or fix in this pr

Will try update in this PR, any obvious place in docs I miss ? ( I only see `SET_STATEMENT`)

File: src/cmd/sql_cmd_test.cc [hunk 1]
```diff
@@ -3935,7 +3935,7 @@ TEST_P(DBSDKTest, GlobalVariable) {
                           {"enable_trace", "false"},
                           {"sync_job", "false"},
                           {"job_timeout", "20000"},
-                          {"execute_mode", "offline"}},
```
Comment (aceforeverd) [inline src/cmd/sql_cmd_test.cc:3954]: recheck

Comment (aceforeverd) [reply src/cmd/sql_cmd_test.cc:3954]: fine, session variable

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**